assets/vue/components/layout/TopbarLoggedIn.vue

```diff
@@ -52,6 +52,8 @@ import Avatar from "primevue/avatar";
 import Menu from "primevue/menu";
 import { usePlatformConfig } from "../../store/platformConfig";
 
+import headerLogoPath from "../../../../assets/css/themes/chamilo/images/header-logo.svg";
```

Comment:
This solves the problem when using dev-server that did not show chamilo header icon